scipy/stats/setup.py

```diff
@@ -27,9 +32,17 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('mvn',
         sources=['mvn.pyf','mvndst.f'],
     )
-        
-    # What do I add here to build Rectangle?
-    
+
+    # Build cython rectangle demo
+    config.add_data_files('rect.pxd')
+    ext = config.add_extension(
+        'rect',
+        sources=['rect.cxx'],
```

Comment:
Cython by default generates sources with .cpp extensions, but scipy renames them to .cxx.  Be careful of overwritting your original files!  (in this case real source is named Rectangle.cpp, so you're safe)

Reply:
You mean if the file were named `XYX.cxx`, it would be overritten when `XYZ.pyx` got compiled?

Reply:
Yes. [Here](https://github.com/scipy/scipy/blob/e777eb9e4a4cd9844629a3c37b3e94902328ad0b/tools/cythonize.py#L284) is the script that actually does this